vmm/src/vm.rs

```diff
@@ -1968,6 +2014,9 @@ impl Vm {
         current_state.valid_transition(new_state)?;
 
         // Load kernel if configured
+        #[cfg(feature = "tdx")]
+        let entry_point = None;
+        #[cfg(not(feature = "tdx"))]
```

Comment:
This doesn't make sense to me. You can still build with the "tdx" feature and use it for a classic VM.

Reply:
Fixed!